js/ngDialog.js

```diff
@@ -869,6 +872,31 @@
                     }
                 );
 
+                // Listen to navigation events to close dialog
+                var uiRouterVersion = privateMethods.detectUIRouter();
+                if (uiRouterVersion === '1.0.0+') {
```

Comment:
Alright, sounds good to me. I'll change them into constant strings and commit again tomorrow. Sorry for the delay, I've been a little busy this few days.